softwerkskammer/lib/groupsAndMembers/groupsAndMembersService.js

```diff
@@ -102,6 +102,18 @@ module.exports = {
     });
   },
 
+  getOrganizersOfGroup: function (groupId, callback) {
+    this.getGroupAndMembersForList(groupId, (error, group) => {
+      if (error) {return callback(error);}
+      if (group === undefined) {
+        return callback(null, []);
+      }
+      const groupMembers = group.members || [];
+      const organizers = (groupMembers).filter(member => group.isOrganizer(member.id()));
```

Comment:
I think one can omit the parentheses:
```suggestion
      const organizers = groupMembers.filter(member => group.isOrganizer(member.id()));
```

Reply:
done